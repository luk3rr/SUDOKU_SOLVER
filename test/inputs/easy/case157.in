090000106 023809000 058000020 000000000 007900804 380600097 002013040 006270000 030000000 
