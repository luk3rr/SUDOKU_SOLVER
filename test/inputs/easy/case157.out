Solving the following grid:
0 9 0 | 0 0 0 | 1 0 6 
0 2 3 | 8 0 9 | 0 0 0 
0 5 8 | 0 0 0 | 0 2 0 
------+-------+------
0 0 0 | 0 0 0 | 0 0 0 
0 0 7 | 9 0 0 | 8 0 4 
3 8 0 | 6 0 0 | 0 9 7 
------+-------+------
0 0 2 | 0 1 3 | 0 4 0 
0 0 6 | 2 7 0 | 0 0 0 
0 3 0 | 0 0 0 | 0 0 0 


Solution found :')

7 9 4 | 3 2 5 | 1 8 6 
1 2 3 | 8 6 9 | 4 7 5 
6 5 8 | 1 4 7 | 9 2 3 
------+-------+------
5 4 9 | 7 8 2 | 3 6 1 
2 6 7 | 9 3 1 | 8 5 4 
3 8 1 | 6 5 4 | 2 9 7 
------+-------+------
9 7 2 | 5 1 3 | 6 4 8 
4 1 6 | 2 7 8 | 5 3 9 
8 3 5 | 4 9 6 | 7 1 2 

Algorithm: BFS
Total time: 81 ms
Total expanded states: 12513
